src/Tizen.Wearable.CircularUI.Forms.Renderer/ContentPopupRenderer.cs

```diff
@@ -0,0 +1,143 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using Tizen.Wearable.CircularUI.Forms;
+using Tizen.Wearable.CircularUI.Forms.Renderer;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+using XForms = Xamarin.Forms.Forms;
+
+[assembly: ExportRenderer(typeof(ContentPopup), typeof(ContentPopupRenderer))]
+
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class ContentPopupRenderer : IContentPopupRenderer
+    {
+        ElmSharp.Popup _popup;
+        ContentPopup _element;
+
+        public void SetElement(Element element)
+        {
+            element.Parent = Application.Current.Parent;
+            element.PropertyChanged += OnElementPropertyChanged;
+
+            _element = element as ContentPopup;
+
+            UpdateContent();
+            UpdateIsShow();
+        }
+
+        public ContentPopupRenderer()
+        {
+            _popup = new ElmSharp.Popup(XForms.NativeParent);
+            _popup.Style = "circle";
+
+            _popup.BackButtonPressed += OnBackButtonPressed;
+            _popup.Dismissed += OnDismissed;
+        }
+
+        ~ContentPopupRenderer()
+        {
+            Dispose(false);
+        }
+
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        public void Dismiss()
+        {
+            _popup?.Hide();
+            _popup?.Dismiss();

```

Comment:
Should not use `Dismiss`, because it destroy popup

Reply:
fixed.